### PR TITLE
Fe checkout form

### PR DESCRIPTION
**Pull request by MonCaptain**
- checkkout form works
- added a toast upon checkout to work as a feedback notification.

### CODE REVIEW

File: api/controllers/order.js [hunk 1]
```diff
@@ -23,10 +23,10 @@ router.get("/myOrders", passport.isAuthenticated(), (req, res) => {
 router.post("/createOrder", passport.isAuthenticated(), (req,res) =>{
     let content = req.body;
     let user = req.user;
-
+    console.log(content)
```
Comment (dwang312) [inline api/controllers/order.js]: Remember to remove this when done being used.

**Review by dwang312 — APPROVED** [review]
LGTM

**Review by MonCaptain — COMMENTED** [review]
console.log removed

File: api/controllers/order.js (hunk 1 above)
Comment (MonCaptain) [reply api/controllers/order.js]: I just did, thank you

**Pull request merged**